src/graphql/Agent/FlowRuns.gql

```diff
@@ -0,0 +1,17 @@
+query AgentSubmittableRuns {
+    flow_run (where: {state: {_eq: "Scheduled"}}) {
+    id
+    name
+    state
+    labels
+    scheduled_start_time
+    version
+    flow {
+      id
+      name
+      schedule
+      environment
+      is_schedule_active
+  }
```

Comment:
Since we have all flows in the data store, this query would be a lot more performant by grabbing the `flow_id` and tying the data together instead